pub-critic/src/app/api/FollowApi.ts

```diff
@@ -43,3 +43,25 @@ export const toggleFollow = async (userId: number) => {
     console.error(error);
   }
 };
+
+export const getUserFollowers = async (userId: number) => {
+  try {
+    const response = await fetch(`${baseURL}/followers/${userId}/followers`, {
+      cache: "no-store",
+    });
+    return response.json();
+  } catch (error) {
+    console.error(error);
+  }
```

Comment:
If you get an error you will return undefined and when you access the response you'll get "accessing props of undefined"
Maybe return a null?

Reply:
:white_check_mark: